pkg/mimir/modules.go

```diff
@@ -185,7 +185,7 @@ func (t *Mimir) initVault() (services.Service, error) {
 		return nil, nil
 	}
 
-	v, err := vault.NewVault(t.Cfg.Vault)
+	v, err := vault.NewVault(t.Cfg.Vault, util_log.Logger, prometheus.WrapRegistererWithPrefix("cortex_", t.Registerer))
```

Comment:
Let's move `cortex_` prefix directly to metric name. It makes it easier to search for metric name in the codebase.